piton/tools/src/proto/vivado/gen_project.tcl

```diff
@@ -94,11 +98,11 @@ foreach inc_file $ALL_INCLUDE_FILES {
 foreach impl_file $ALL_RTL_IMPL_FILES {
     if {[file exists $impl_file]} {
         set file_obj [get_files -of_objects $fileset_obj [list "$impl_file"]]
-        if {[file extension $impl_file] == ".sv"} {
```

Comment:
Can we change this one to check BP? Before with ariane we only needed to set .sv files as systemverilog

Reply:
Removed! Actually not needed for BP